apps/backend/app.py

```diff
@@ -185,12 +199,41 @@ def upload_experiment_results(expId, trialExtraFile, postProcess):
     uploadBlob = firebaseBucket.blob(f"results/result{expId}.csv")
     uploadBlob.upload_from_filename('results.csv')
 
+    # Upload to MongoDB
+    try:
+            mongoClient.admin.command('ping')
+    except ConnectionFailure as err:
+        raise GladosInternalError("MongoDB server not available/unreachable") from err
+        
+    
+    print('Uploading to MongoDB')
+    experimentFile = open(f"results.csv") # there is probably a better way to do this
+    experimentData = experimentFile.read()
+    experimentFile.close()
+    experimentResultEntry = {"_id": expId,
+                        "resultContent": experimentData}
+    try:
+        resultId = mongoResultsCollection.insert_one(experimentResultEntry).inserted_id
+        print(f"inserted result csv into mongodb with id: {resultId}")
+    except Exception as err:
+        raise GladosInternalError("Encountered error while inserting results into MongoDB") from err
     if trialExtraFile != '' or postProcess:
         print('Uploading Result Csvs')
         try:
             shutil.make_archive('ResultCsvs', 'zip', 'ResCsvs')
             uploadBlob = firebaseBucket.blob(f"results/result{expId}.zip")
             uploadBlob.upload_from_filename('ResultCsvs.zip')
+            
+            # mongoResultsZipCollections
+            with open("ResultCsvs.zip", "rb") as file:
+                encoded = Binary(file.read())
+            experimentZipEntry = {"_id": expId,
+                            "fileContent": encoded}
+            try:
+                resultZipId = mongoResultsZipCollections.insert_one(experimentZipEntry).inserted_id
```

Comment:
```suggestion
                resultZipId = mongoResultsZipCollections.insert_one(experimentZipEntry).inserted_id # TODO store this id in the firebase entry somewhere so it can be accessed by the frontend
```

Reply:
the id is the same as the experiment id we normally use, i used the _id field to make it work as the mongoDB key as well